TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datahandlers/NettyPacketToHttpConsumer.java

```diff
@@ -227,7 +239,11 @@ private void deactivateChannel() {
                     log.atDebug().setMessage(()->"Ignoring an exception that the "+handlerName+" wasn't present").log();
                 }
             }
-            while (!(pipeline.last() instanceof SslHandler) && (pipeline.last() != null)) {
+            while (true) {
```

Comment:
Did we intend to remove the `&& (pipeline.last() != null)` condition?

Reply:
Yes, the ConnectionClosedListenerHandler handler will ALWAYS be present at the head of the pipeline now.  Before, we could have either SSL or no handler, but the ConnectionClosedListenerHandler is always applicable and should be carried forward until the connection is closed (at which point, the pipeline will be torn down)